doc/layers.md

```diff
@@ -0,0 +1,92 @@
+# Layers
+
+Layers are an useful feature that lets you group renderization by context, and to also to obtain a way of pre rendering things, enabling for example the renderization in memory of parts of your game that don't change much, like a background for example, and by doing that, freeing resources for more dynamic content that needs to be rendered every loop cycle.
+
+There are two types of layers on Flame: `DynamicLayer` and `PreRenderedLayer`.
+
+## DynamicLayer
+
+Dynamic layers are layers that are rendered every time that they are draw on the canvas, as the name suggests, it is meant for dynamic content and is most useful to group renderizations that are of the same context.
```

Comment:
> canvas, as the name suggests, it is meant  

I think either

canvas. As the name suggests, it is meant 

or 

canvas, as the name suggests. It is meant 

(depending on what you meant) would be better